server/src/visitors/YmlFunctionVisitor.ts

```diff
@@ -1,11 +1,29 @@
 import { CompletionItemKind } from "vscode-languageserver";
 import { YmlCompletionItemsProvider } from "../completion/YmlCompletionItemsProvider";
 import { YmlDefinitionProvider } from "../definitions";
-import { FunctionContext } from "../grammar";
+import {
+  FunctionContext,
+  MandatoryArgDeclContext,
+  MemberDeclarationContext,
+  VariableBlockContentContext,
+} from "../grammar";
 import { createLocation, createNewCompletionItem } from "./VisitorsUtils";
 import { YmlBaseVisitor } from "./YmlBaseVisitor";
 
 export class YmlFunctionVisitor extends YmlBaseVisitor {
+  /**
+   * First character of the Function object.
+   * Will be used as the start of the scope for its arguments and local variables.
+   */
```

Comment:
Silly question again: Why are there similar pieces of information here and in the `ICompletionItemLocation` thingy from above?

Reply:
This duplicated elements will be removed when a real model be created.